src/components/activity-header/header.scss

```diff
@@ -26,19 +26,19 @@
       align-items: center;
       justify-content: flex-start;
       height: 100%;
-      cursor: pointer;
       &.no-link {
         cursor: auto;
       }
       .logo {
         height: 80%;
         background-color: white;
+        cursor: pointer;
       }
       .project-logo {
         height: 80%;
         background-color: white;
         padding: 6px 10px;
-
+        cursor: pointer;
```

Comment:
Should all project logos have a pointer?  Precipitating change (use `"project_id": 11,` in an activity JSON to see it in use) doesn't have an explicit link.  However, right now we're linking to the CC site.  Is that correct?  So this would be the current rules set:
- if the project specifies a logo, use the project logo
- if the project doesn't specify a logo, use the CC logo
- if the project specifies a URL, link to the URL from the logo
- if the project doesn't specify a logo, link to the CC site (which means a project might HAVE a logo like a "Global EduCorp Limited" log, but if it DOESN'T have a URL specified, the logo will open the CC site when clicked.  Is that right?).

Reply:
Good catch, I forgot that there may be a project logo but not a link associated with it. I was thinking it was always going to be a project or CC logo.

Reply:
Ended up pulling the logo code out into its own component since there was other logic and css I needed to add for tabbing to it. And then maybe we can also use it for the logos in the footer.

Reply:
Yeah, I wasn't sure what exactly was the  "correct" thing to do here.